python/inline/conftest.py

```diff
@@ -470,10 +497,21 @@ def parse_constructor(self, node):
                     # check if "disabled" is a boolean
                     elif (
                         index == 4
-                        and isinstance(arg, ast.NameConstant)
-                        and isinstance(arg.value, bool)
+                        and isinstance(arg, ast.Num)
+                        and isinstance(arg.value, int)
                     ):
-                        self.cur_inline_test.disabled = arg.value
+                        self.cur_inline_test.timeout = arg.value
+                    # check if "timeout" is a positive integer
+                    elif (
+                        index == 5
+                        and isinstance(arg, ast.Num)
+                        and isinstance(arg.n, int)
```

Comment:
It is better to use type "float" than "int" for timeout value

Reply:
Fixed